### PR TITLE
Amazon locations example

### PR DESCRIPTION
**Pull request by menusal**
## Description

Replicate the following example: [codesandbox](https://codesandbox.io/s/carto-deck-gl-examples-rimo9x?from-embed=&initialpath=/styling/proportional-symbol-map.html&file=/styling/proportional-symbol-map.html) in Amazon locations

image

Bonus: add a slider control to filter by population
Interactivity as hover is required to show the name + pop

Shorcut: [sc-361050](https://app.shortcut.com/cartoteam/story/361050/basemaps-basic-hello-world)

## CHECKLIST

- [x] Amazon locations base map
- [x] Tooltip
- [x] Slider to filter by population

## TODO

- Update attribution info
- Basemap selector

### CODE REVIEW

File: amazon-locations/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "carto-deckgl-example-basemaps",
```
Comment (alasarr) [inline amazon-locations/package.json:2]: amazon-basemap

File: amazon-locations/.env [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# API Base URL (copy this from CARTO Workspace -> Developers)
+VITE_API_BASE_URL=https://gcp-us-east1.api.carto.com
+# Connection ID (app.carto.com -> Developers -> Manage API Access Tokens)
+VITE_API_CONNECTION_NAME=bqconn2
```
Comment (alasarr) [inline amazon-locations/.env:4]: Apply same things than https://github.com/CartoDB/deck.gl-examples/pull/1

File: amazon-locations/index.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    style: `https://maps.geo.${region}.amazonaws.com/maps/v0/maps/${mapName}/style-descriptor?key=${apiKey}`,
+    interactive: false
+  });
+
+  map.addControl(new maplibregl.NavigationControl({ showCompass: false }), "bottom-right");
+  return map;
+}
+
+function setupEventHandlers(deckInstance, mapInstance) {
+  if (!selectedPopulationSelector || !populationLabel) {
```
Comment (alasarr) [inline amazon-locations/index.ts:57]: populationLabel is not used

File: amazon-locations/index.ts [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+const getPointRadius = d => (POPULATION_RADIUS_SCALE * Math.sqrt(d.properties.pop_2015));
+
+function initializeMap(apiKey, mapName, region) {
+  const map = new maplibregl.Map({
+    container: 'map',
+    style: `https://maps.geo.${region}.amazonaws.com/maps/v0/maps/${mapName}/style-descriptor?key=${apiKey}`,
+    interactive: false
+  });
+
+  map.addControl(new maplibregl.NavigationControl({ showCompass: false }), "bottom-right");
```
Comment (alasarr) [inline amazon-locations/index.ts:52]: Remove it to stay homogeneous with the rest of examples

**Review by alasarr — CHANGES_REQUESTED** [review]

File: amazon-locations/package.json (hunk 1 above)
Comment (menusal) [reply amazon-locations/package.json:2]: Done

File: amazon-locations/index.ts [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+import { VectorTileLayer } from '@deck.gl/carto';
+import { colorCategories, vectorQuerySource } from '@deck.gl/carto';
+
+const cartoConfig = {
+  apiBaseUrl: import.meta.env.VITE_API_BASE_URL,
+  accessToken: import.meta.env.VITE_API_ACCESS_TOKEN,
+  connectionName: import.meta.env.VITE_API_CONNECTION_NAME
+};
+
+const SQL_QUERY = `
```
[the remark this answers is not in the document]
Comment (menusal) [reply amazon-locations/index.ts:14]: Done

File: amazon-locations/index.ts [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+const INITIAL_VIEW_STATE = { latitude: 15, longitude: 10, zoom: 2 };
+const MIN_POPULATION = 1;
+const MAX_POPULATION = 800000000;
+const POPULATION_RADIUS_SCALE = 30.0 / 27620.2642999664;
+
+let selectedMaxPopulation = MAX_POPULATION;
+const selectedPopulationSelector = document.getElementById('population-slider');
+const populationLabel = document.getElementById('slider-value');
+
+const getColorConfiguration = () => colorCategories({
```
[the remark this answers is not in the document]
Comment (menusal) [reply amazon-locations/index.ts:30]: Done

File: amazon-locations/package.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "carto-deckgl-amazon-locations-example",
```
Comment (alasarr) [inline amazon-locations/package.json:2]:
```suggestion
  "name": "carto-deckgl-amazon-locations",
```

**Review by alasarr — APPROVED** [review]

**Pull request merged**